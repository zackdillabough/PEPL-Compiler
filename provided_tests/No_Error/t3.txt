POLY T(X, Y) = X + Y;
POLY S(R, Y, U, W) = R^2 + Y^2 + W^44 + U;
POLY Y = x + 5555;
START
INPUT A;
INPUT B;
INPUT C;
Y(A);
T(B, C);
S(A, B, C, 0);
1 2 3 4 5
