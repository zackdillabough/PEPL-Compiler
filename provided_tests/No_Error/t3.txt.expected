5556
5
8
